webapp/src/pages/UserPage/UserPage.tsx

```diff
@@ -152,6 +165,17 @@ export default function UserPage() {
           undefined
         }
         secondaryButton={
+          (!overridePermissions &&
+            user &&
+            user.isFriend !== null &&
+            userStatus(user.id) === 'playing' && {
+              variant: ButtonVariant.ALTERNATIVE,
+              iconVariant: IconVariant.PLAY,
+              onClick: () => {
+                socket.emit('getPlayerGame', { playerId: user.id });
+              },
+              children: 'spectate player',
```

Comment:
What do you think about 'watch live game' instead of 'spectate game'?

Reply:
I've updated it. Thanks for the suggestion.